plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -400,6 +401,22 @@ public void doExport(StaplerRequest req, StaplerResponse res) throws Exception {
         export(res.getOutputStream());
     }
 
+    /**
+     * Export JSONSchema to URL
+     * @throws Exception
+     */
+    @RequirePOST
+    public void dov2Schema(StaplerRequest req, StaplerResponse res) throws Exception {
```

Comment:
1) this isn't linked to from anywhere, is that intentional?

2) does this need to be require post? from a rest api design it doesn't really make sense and the previous schema was available as a GET
cc @oleg-nenashev

Reply:
I Tried using Postman and it still does not work :(

Reply:
it works for me, `http://localhost:8080/jenkins/configuration-as-code/v2Schema`

Reply:
Thats great then, we are almost done with this PR

Reply:
> 1. this isn't linked to from anywhere, is that intentional?
> 2. does this need to be require post? from a rest api design it doesn't really make sense and the previous schema was available as a GET
>    cc @oleg-nenashev

@timja We could make it a GET